css/style.css

```diff
@@ -149,3 +149,555 @@ section .content p {
     margin-left: 45px;
   }
 }
+
+.section-1 .first-header h2 {
+  position: static;
+  width: 327px;
+  height: 97px;
+  font-size: 40px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 60rem;
+  margin-left: 2.1rem;
+  line-height: 52px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.section-1 .first-header span {
+  display: flex;
+  justify-content: center;
+  width: 10%;
+  height: auto;
+  border-bottom: 4px solid #36b37e;
+  border-radius: 5px;
+  text-align: center;
+  margin-left: 45%;
+}
+
+.section-1 .first-project {
+  position: static;
+  left: 0;
+  right: 0.6%;
+  top: 0.13%;
+  bottom: 83.92%;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] Please try to use correctly the flex inside the work section in order to layout the elements. Try to use it for other projects also as mentioned by the previous reviewer.